opentelekomcloud/provider.go

```diff
@@ -373,6 +373,7 @@ func Provider() terraform.ResourceProvider {
 			"opentelekomcloud_s3_bucket_policy":                   s3.ResourceS3BucketPolicy(),
 			"opentelekomcloud_s3_bucket_object":                   s3.ResourceS3BucketObject(),
 			"opentelekomcloud_sfs_file_system_v2":                 sfs.ResourceSFSFileSystemV2(),
+			"opentelekomcloud_sfs_turbo_v1":                       sfs.ResourceSFSTurboV1(),
```

Comment:
Let's name it `opentelekomcloud_sfs_turbo_share_v1`, or something like that. Otherwise, it's not clear what is actually managed by this resource.